remotespark/magics/sparkmagicsbase.py

```diff
@@ -43,12 +43,15 @@ def __init__(self, shell, data=None):
 
         self.logger.debug("Initialized spark magics.")
 
-    def execute_against_context_that_returns_df(self, method, cell, session, output_var):
+    def execute_against_context_that_returns_df(self, method, cell, session, output_var, quiet):
         try:
             df = method(cell, session)
             if output_var is not None:
                 self.shell.user_ns[output_var] = df
-            return df
+            if quiet:
```

Comment:
No. You can run SQL queries which have a side effect and which return no value (e.g. CREATE TABLE)